espei/paramselect.py

```diff
@@ -973,102 +397,19 @@ def bin_int_sort_key(x):
                     keys_to_remove.append(key)
             for key in keys_to_remove:
                 parameters.pop(key)
-        print(degree_polys)
+        logging.debug('Polynomial coefs: {}'.format(degree_polys))
         # Insert into database
-        symmetric_interactions = _generate_symmetric_group(interaction,
-                                                           symmetry)
+        symmetric_interactions = _generate_symmetric_group(interaction, symmetry)
         for degree in np.arange(degree_polys.shape[0]):
             if degree_polys[degree] != 0:
                 for syminter in symmetric_interactions:
-                    dbf.add_parameter('L', phase_name,
-                                      tuple(map(_to_tuple, syminter)), degree,
-                                      degree_polys[degree])
-    # Now fit ternary interactions
-
+                    dbf.add_parameter('L', phase_name, tuple(map(_to_tuple, syminter)), degree, degree_polys[degree])
+    # TODO: fit ternary interactions
     if hasattr(dbf, 'varcounter'):
```

Comment:
Yes, you are correct. I had in mind it was for something else. Definitely should be kept.